### PR TITLE
Various fixes and rate limiting information X-headers

### PR DESCRIPTION
**Pull request by bravo-kernel**
Relates to #10 and foobarred #11

### CODE REVIEW

**Comment by bravo-kernel** [comment]
Good to merge if you ask me.

File: tests/.travis.ini [hunk 1]
```diff
@@ -0,0 +1,3 @@
+extension=apc.so
```
Comment (jadb) [inline tests/.travis.ini]: Maybe add a file doc block to explain what it is here for?

File: .travis.yml [hunk 2]
```diff
@@ -32,6 +32,8 @@ before_script:
   - composer self-update
   - composer install --prefer-dist --no-interaction
 
+  - phpenv config-add ./tests/.travis.ini
```
Comment (jadb) [inline .travis.yml:35]: :+1:

**Comment by bravo-kernel** [comment]
INI docblock https://github.com/bravo-kernel/Throttle/commit/a68e51337772954f73f9d3cbe2967560aefafd1c#diff-766c41a4683e609eab9e45c54f73032eR1

**Comment by jadb** [comment]
Very good job on that one! :+1:

**Comment by bravo-kernel** [comment]
thanks

**Pull request merged**